### PR TITLE
SSO: rename module to Secure Sign On

### LINKED ISSUE #5957: SSO: rename module to Secure Sign On
We've started to rename "Single Sign On" into "Secure Sign On" in many areas of Jetpack, but it would be nice to make sure everything is updated everywhere where we mention the module.

### PR DESCRIPTION
**Pull request by jom**
Fixes #5957

#### Changes proposed in this Pull Request:
Rename **Single Sign On** to **Secure Sign On**

#### Testing instructions:

* Verify all instances of "Single Sign On" have been properly renamed

#### Proposed changelog entry for your changes:
Renamed 'Single Sign On' module to 'Secure Sign On'

### CODE REVIEW

File: modules/module-headings.php [hunk 1]
```diff
@@ -1,229 +1,229 @@
...
- *
- * @param string $key Module file name without .php
- *
- * @return array
- */
-function jetpack_get_module_i18n( $key ) {
-	static $modules;
-	if ( ! isset( $modules ) ) {
+<?php
+// Do not edit this file. It's generated by jetpack/tools/build-module-headings-translations.php
```
Comment (dereksmart) [inline modules/module-headings.php]: I'm confused how/why this file changed.  Did you run the module-headings build?

Comment (jom) [inline modules/module-headings.php]: Hi @dereksmart! I ran `jetpack/tools/build-module-headings-translations.php` after changing the name in `modules/sso.php`. If this happens at another time in the release cycle, I can revert.

Comment (dereksmart) [inline modules/module-headings.php]: @jom yup! We'll take care of this later in the cycle.  You can go ahead and revert this file

**Comment by jom** [comment]
@dereksmart I've reverted the changes to the `modules/module-headings.php` file.

**Comment by jeherve** [comment]
This is being discussed in p7kMJG-19T-p2.

**Comment by dereksmart** [comment]
Hey @jom thanks for this patch, however after discussing internally we're going to go in a different direction.  If you could, please leave this branch open in your fork as @rickybanister and @ebinnion may want to cherry-pick some of your changes over to an internal branch.  

Thanks!

**Pull request closed without merging**